### ISSUE TITLE
filtering out data while exporting user accounts from admin dashboard

### ISSUE THREAD

**Issue by saga2405** [llaske/sugarizer-server#313]
the button `export users` in the admin dashboard(`http://127.0.0.1:8080/dashboard/users`) exports all types of user accounts in the form of CSV file regardless of the type(`student, teacher, admin`) we've selected
the downloaded file includes all three types of accounts - admin, teacher, student
We can filter out the data and export only those type of account which is selected in the query 
for example: If I'm viewing student accounts and click on export users - the downloaded csv file should only contain users of type student, the same goes for teacher and admin type
should i fix this?

**Comment by saga2405** [comment]
@llaske can you please give your inputs

**Comment by llaske** [comment]
You're right it make sense to use filter (both user type and user name) in filtering.

**Comment by saga2405** [comment]
@llaske @NikhilM98 I've made a request for this. can you please check

**Comment by parteekcoder** [comment]
hey @NikhilM98 @llaske Can I work upon this

**Comment by NikhilM98** [comment]
@parteekcoder There is someone already working on this issue. Let's wait for them to respond.
